systemtests/server/common.ts

```diff
@@ -0,0 +1,140 @@
+/**
```

Comment:
Some of these are gonna show up as doc comments? Doesn't matter too much as its in tests.

Reply:
Cool, sorted for consistency